src/huggingface_hub/hf_api.py

```diff
@@ -1341,7 +1375,7 @@ def list_models(
 
         >>> # List only models from the AllenNLP library
         >>> api.list_models(filter="allennlp")
-
+        ```
```

Comment:
~~intended?~~ yes i guess so

Reply:
Yes I just realized [docs](https://huggingface.co/docs/huggingface_hub/v0.19.1/en/package_reference/hf_api#huggingface_hub.HfApi.list_models) looks awful at the moment :smile: 

![Uploading image.png…]()